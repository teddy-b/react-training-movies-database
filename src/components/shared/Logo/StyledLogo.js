import styled from 'styled-components'

import { red } from '../../../constants/styles'

const StyledLogo = styled.p`
  color: ${red};
  font-weight: bold;
`

export default StyledLogo
