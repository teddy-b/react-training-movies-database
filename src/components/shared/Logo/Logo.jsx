/* @flow */

import * as React from 'react'

import StyledLogo from './StyledLogo'

const Logo = (): React.Node => (
  <StyledLogo>netflixroulette</StyledLogo>
)

export default Logo
