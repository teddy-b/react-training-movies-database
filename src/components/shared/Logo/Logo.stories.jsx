import React from 'react'

import { storiesOf } from '@storybook/react'

import Logo from './Logo'

storiesOf('Components/Logo', module)
  .add('default', () => (
    <Logo />
  ))
