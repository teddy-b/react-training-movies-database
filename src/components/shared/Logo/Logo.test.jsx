/* @flow */

import * as React from 'react'

import renderer from 'react-test-renderer'

import Logo from './Logo'

describe('Logo', () => {
  it('renders correctly', () => {
    const component = renderer.create(<Logo />)
    const tree = component.toJSON()

    expect(tree).toMatchSnapshot()
  })
})
