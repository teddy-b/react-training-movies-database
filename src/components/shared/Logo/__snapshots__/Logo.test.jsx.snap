// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders correctly 1`] = `
<p
  className="sc-bdVaJa hBMEvi"
>
  netflixroulette
</p>
`;
